src/actions/motion.ts

```diff
@@ -513,7 +516,11 @@ class MoveRight extends BaseMovement {
   keys = ['l'];
 
   public async execAction(position: Position, vimState: VimState): Promise<Position> {
-    return new Position(position.line, position.character + 1);
```

Comment:
I am not 100% sure that this change is correct. It works for the motion test cases below, and I think it is functionally equivalent in the non-whichwrap-enabled case, but someone with more knowledge should take a look.

Reply:
I saw this recently as well, your change makes sense to me either way, if it causes issues we can fix them